### ISSUE TITLE
support es module?

### ISSUE THREAD

**Issue by deyihu** [Turfjs/turf#1772]
Please provide the following when reporting an issue:

- [ ] GeoJSON data as a [gist file](https://gist.github.com/) or [geojson.io](http://geojson.io/) (filename extension must be `.geojson`).
- [ ] Snippet of source code or for complex examples use [jsfiddle](https://jsfiddle.net/).

```js
import hexGrid from "@turf/hex-grid";
import { GeoJSON } from "YY";


export function grid(bbox, cellSize) {
    const geojson = hexGrid(bbox, cellSize / 1000);
    return GeoJSON.toGeometry(geojson);
}

```

If I use webback for packaging, that's good，

**But I used rollup for packaging, and unfortunately it happened.**

![20191021112419](https://user-images.githubusercontent.com/25998927/67174656-6cd5d180-f3f5-11e9-89a4-09127324ad7f.png)

![20191021112720](https://user-images.githubusercontent.com/25998927/67174735-cdfda500-f3f5-11e9-9478-39debd7cdfc1.png)

**Comment by rowanwins** [comment]
Perhaps try the v7 alpha release
https://www.npmjs.com/package/turf/v/7.0.0-alpha.1

**Comment by rowanwins** [comment]
Resolved with v6.2.0 release
